src/components/pages/Incidents/Map/Map.js

```diff
@@ -12,6 +13,11 @@ const Map = props => {
     lat: 38,
     lng: 267,
   };
+
+  const mapOptions = {
+    fullscreenControl: false,
```

Comment:
Great move changing the opacity of the map.  Helps fit the vision of our project.  great idea using snazzymapGreyScale.js in a different file instead of inline.